src/hyp3_srg/time_series.py

```diff
@@ -0,0 +1,325 @@
+        baselines: tuple containing the time baseline and spatial baseline
+        dem_shape: tuple containing the dem width and dem length
+        work_dir: the directory containing the GSLCs
+    """
+    dem_width, dem_length = dem_shape
+    looks_down, looks_across = looks
+    time_baseline, spatial_baseline = baselines
+
+    utils.call_stanford_module('sentinel/sbas_list.py', args=[time_baseline, spatial_baseline], work_dir=work_dir)
+
+    sbas_args = ['sbas_list', '../elevation.dem.rsc', 1, 1, dem_width, dem_length, looks_down, looks_across]
+    utils.call_stanford_module('sentinel/ps_sbas_igrams.py', args=sbas_args, work_dir=work_dir)
+
+
+def unwrap_interferograms(dem_shape: tuple[int], unw_shape: tuple[int], work_dir: Path) -> None:
+    """Unwraps wrapped interferograms in parallel
+
+    Args:
+        dem_shape: tuple containing the dem width and dem length
+        unw_shape: tuple containing the width and length from the dem.rsc file
+        work_dir: the directory containing the wrapped interferograms
+    """
+    dem_width, dem_length = dem_shape
+    unw_width, unw_length = unw_shape
+
+    reduce_dem_args = ['../elevation.dem', 'dem', dem_width, dem_width // unw_width, dem_length // unw_length]
+    utils.call_stanford_module('util/nbymi2', args=reduce_dem_args, work_dir=work_dir)
+    utils.call_stanford_module('util/unwrap_parallel.py', args=[unw_width], work_dir=work_dir)
+
+
+def compute_sbas_velocity_solution(
+    threshold: float, do_tropo_correction: bool, unw_shape: tuple[int], work_dir: Path
+) -> None:
+    """Computes the sbas velocity solution from the unwrapped interferograms
+
+    Args:
+        threshold: ...
+        do_tropo_correction: ...
+        unw_shape: tuple containing the width and length from the dem.rsc file
+        work_dir: the directory containing the wrapped interferograms
+    """
+    unw_width, unw_length = unw_shape
+
+    utils.call_stanford_module('sbas/sbas_setup.py', args=['sbas_list', 'geolist'], work_dir=work_dir)
+    copyfile(work_dir / 'intlist', work_dir / 'unwlist')
+    utils.call_stanford_module('util/sed.py', args=['s/int/unw/g', 'unwlist'], work_dir=work_dir)
+
+    num_unw_files = 0
+    num_slcs = 0
+    with open(work_dir / 'unwlist', 'r') as unw_list:
+        num_unw_files = len(unw_list.readlines())
+    with open(work_dir / 'geolist', 'r') as slc_list:
+        num_slcs = len(slc_list.readlines())
+
+    ref_point_args = ['unwlist', unw_width, unw_length, threshold]
+    utils.call_stanford_module('int/findrefpoints', args=ref_point_args, work_dir=work_dir)
+
+    if do_tropo_correction:
+        tropo_correct_args = ['unwlist', unw_width, unw_length]
+        utils.call_stanford_module('int/tropocorrect.py', args=tropo_correct_args, work_dir=work_dir)
+
+    sbas_velocity_args = ['unwlist', num_unw_files, num_slcs, unw_width, 'ref_locs']
+    utils.call_stanford_module('sbas/sbas', args=sbas_velocity_args, work_dir=work_dir)
```

Comment:
```suggestion
    ref_point_args = ['unwlist', unw_width, unw_length, threshold]
    utils.call_stanford_module('int/findrefpoints', args=ref_point_args, work_dir=work_dir)

    if do_tropo_correction:
        tropo_correct_args = ['unwlist', unw_width, unw_length]
        utils.call_stanford_module('int/tropocorrect.py', args=tropo_correct_args, work_dir=work_dir)

    num_unw_files = 0
    with open(work_dir / 'unwlist', 'r') as unw_list:
        num_unw_files = len(unw_list.readlines())

    num_slcs = 0
    with open(work_dir / 'geolist', 'r') as slc_list:
        num_slcs = len(slc_list.readlines())
 
    sbas_velocity_args = ['unwlist', num_unw_files, num_slcs, unw_width, 'ref_locs']
    utils.call_stanford_module('sbas/sbas', args=sbas_velocity_args, work_dir=work_dir)
```
I think it's cleaner written like this. We don't use `num_unw_files` and `num_slcs` until the sbas call.